module/src/components/singleSelect/singleSelect.component.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import Select, { GroupBase, OnChangeValue } from "react-select";
+import SelectRef from "react-select/dist/declarations/src/Select";
+import { Form } from "../../hooks";
+import { IIcon, IconSet } from "../icon";
+import { ValidationErrors } from "../validationErrors";
+
+export type ReactSingleSelectRef = SelectRef<
+  ISelectOptionType<string>,
+  false,
+  GroupBase<ISelectOptionType<string>>
+>;
+
+export type ISelectOptionType<TSelectData = any> = {
+  value: TSelectData;
+  label: string;
+};
+
+export interface ISingleSelectProps<TSelectData = string> {
+  /**  prop for binding to an Armstrong form binder (see forms documentation) */
+  bind: Form.IBindingProps<string>;
+
+  /** the options to be displayed in the input */
+  options: ISelectOptionType<TSelectData>[];
+
+  /** adds a label above the input */
+  label?: string;
```

Comment:
Are we bringing this back then? I thought we were leaving labelling up to the consuming app. If we _are_ bringing it back, the type should be `ReactNode` to support custom markup.